hibernate-core/src/main/java/org/hibernate/engine/jdbc/batch/internal/BatchingBatch.java

```diff
@@ -78,7 +78,7 @@ public void addToBatch() {
 		}
 		catch ( SQLException e ) {
 			abortBatch();
-			LOG.debugf( "SQLException escaped proxy", e );
+			LOG.debug( "SQLException escaped proxy", e );
```

Comment:
The bug here is the exception will end up as format parameter, not as root cause. Fixed it to align with other logging statements in the same class.